templates/dice-config.hbs

```diff
@@ -18,6 +18,14 @@
         {{/each}}
     </section>
 
+    <section>
+        <div class="form-group">
```

Comment:
I'm not sure this should be a setting. Feels to me like this should just be the default behavior. Do you have use cases in mind where the user would not want to identify inner terms as fulfillable?

Reply:
To be honest, I added this as an opt-in so that the exceptional behaviour would be executed only in pf2e and potentially other cases, to prevent from potential issues in other systems that I had not encountered. If you are happy to make this the default behaviour, I'll gladly make it so.